pantheon-bundle/src/main/java/com/redhat/pantheon/validation/helper/XrefValidationHelper.java

```diff
@@ -8,14 +8,19 @@
 
 public class XrefValidationHelper {
 
-    private static Map<String, List<String>> xRefs = new HashMap<>();
+    private static List<String> xRefs;
 
-    public static List<String> getObjectsToValidate(String uuid) {
-        return xRefs.get(uuid);
+    public static List<String> getObjectsToValidate() {
+        return xRefs;
     }
 
-    public static void setObjectsToValidate(String uuid, List<String> objectsToValidate) {
-        if(objectsToValidate.size()>0)
-            xRefs.put(uuid, objectsToValidate);
+    public static void initList() {
```

Comment:
This might not be required if initialization and declaration are done together.

Reply:
we require list to initialize each time, to clear previous memory if any, for document,